net-mail/fdm/fdm-1.9.ebuild

```diff
@@ -13,9 +13,10 @@ SRC_URI="https://github.com/nicm/fdm/releases/download/${PV}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="courierauth examples pcre"
+IUSE="courierauth examples libressl pcre"
 
-DEPEND="dev-libs/openssl:0
+DEPEND="!libressl? ( dev-libs/openssl:0 )
+	libressl? ( dev-libs/libressl:0 )
```

Comment:
There is a general consensus that we should be doing to protect against abi changes in either pkg.

```
DEPEND="!libressl? ( dev-libs/openssl:0= )
	          libressl? ( dev-libs/libressl:0= )
```